src/OpenTelemetry.Exporter.OpenTelemetryProtocol.Logs/OpenTelemetry.Exporter.OpenTelemetryProtocol.Logs.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
@reyang Following up on our conversation in slack...

I've created the `OpenTelemetry.Exporter.OpenTelemetryProtocol.Logs` project. The only thing it exposes is an extension method.

You seemed to suggest a preference for not creating a new package to simplify publishing and changelog management. The reason we discussed a separate package in this week's SIG meeting is that the logs proto is not yet deemed stable.

Are you thinking that since all we're exposing is an extension method, you'd prefer to keep it in the main package?

Reply:
What you've done here was exactly what I was thinking (I pinged you on Slack because I saw the PR is becoming big - probably due to the rebase and different branch target so I was worried).
Now the PR looks pretty good to me! 💯